lib/puppet/reports/servicenow.rb

```diff
@@ -16,7 +16,16 @@ def process
 
     Puppet.info(sn_log_entry("incident creation conditions: #{incident_creation_conditions}"))
 
-    unless create_incident?(status, resource_statuses, incident_creation_conditions)
+    # Some incident creation conditions depend on the resource events. Thus, we go ahead
```

Comment:
Thanks, could we move this comment over to `calculate_satisfied_conditions` too?

Reply:
sure, done.

Reply:
Did you push up the change? I still see this comment here.

Reply:
whoops, should be pushed up now.